.github/workflows/ci.yml

```diff
@@ -58,3 +58,29 @@ jobs:
 
       - name: Run the unit tests
         run: invoke tests --cloud --unit --timeout=1.5 --slowest=8
+
+  marker-tests:
+    strategy:
+      matrix:
+        markers: ["big", "cli", "filesystem", "project"]
```

Comment:
Initially, I'm only using the markers that I know don't require any extra setup or external dependencies (upping a postgres container for example).

We'll need a mechanism for setting up whatever extra dependencies are required based on this marker but I'm deferring that work for now.